src/redux/rootReduser.js

```diff
@@ -1,7 +1,7 @@
 import { combineReducers } from 'redux';
-import news from './news/news.reduser';
+import News from './news/news.reducer';
```

Comment:
Please, rename it to the newsReducer. Key in store should still be News

Reply:
done